.github/workflows/build.yml

```diff
@@ -2,45 +2,49 @@ name: Build
 
 on:
   push:
-    branches: main
+    branches:
+      - main
   pull_request:
-    branches: '*'
+    branches:
+      - main
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        jupyterlab: ['2.0', '3.0.0']
-      fail-fast: false
     steps:
-      - name: Checkout
+      - name: Checkout 🏷️
         uses: actions/checkout@v2
 
-      - name: Install node
+      - name: Use Node 14 🕹️
         uses: actions/setup-node@v2
         with:
-          node-version: '12.x'
+          node-version: '14.x'
 
-      - name: Install Python
+      - name: Install Python 🐍
         uses: actions/setup-python@v2
         with:
           python-version: '3.6'
           architecture: 'x64'
 
-      - name: Install dependencies
-        run: python -m pip install jupyterlab~=${{ matrix.jupyterlab }}
+      - name: Install dependencies ⚙️
+        run: python -m pip install -U jupyterlab~=3.0 check-manifest
```

Comment:
You don't think it's worth checking multiple 3.x minors?

Reply:
Not for now, no.

If we get a minor version that we **really** need to support, I might reconsider